scripts/get-updates.jenkins.io-trustedci-credentials.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+# We do not want any credential persisted in the repository!
+temp_dir="$(mktemp -d)"
+zip_elements=()
+
+for service in content redirections
+do
+  env_file=.env-"${service}"
+  # The -raw flag ensures there are not heredoc markers in the output - https://developer.hashicorp.com/terraform/cli/commands/output#raw
+  terraform output -raw -no-color update_center_fileshare_env-"${service}" > "${temp_dir}"/"${env_file}"
+  zip_elements+=("${env_file}")
+done
+# terraform output update_center_fileshare_env-redirections > "${temp_dir}/.env-redirections"
```

Comment:
```suggestion
```

Reply:
sorry for the wrong account but it's still me @smerle33